FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -161,7 +166,9 @@ - (FIRTrace *)activeTrace {
  * @return true if the platform could prewarm apps on the current device
  */
 + (BOOL)isPrewarmAvailable {
-  if (![[[GULAppEnvironmentUtil applePlatform] lowercaseString] isEqualToString:@"ios"]) {
+  NSString *platform = [[GULAppEnvironmentUtil applePlatform] lowercaseString];
+  if (![platform isEqualToString:@"ios"] && ![platform isEqualToString:@"maccatalyst"] &&
+      ![platform isEqualToString:@"tvos"]) {
```

Comment:
nit: Can you add a comment? The code is a bit hard to read. Can you do something like:
```
if (iOS && catalyst && tvos) {
  if (version > 15) {
   return YES;
  }
}
return No;
```